python-client/giskard/scanner/report.py

```diff
@@ -98,6 +99,9 @@ def generate_test_suite(self, name=None):
         self._track_suite(suite, name)
         return suite
 
+    def generate_dataset(self):
```

Comment:
Does this belong here? I‘m not sure.

Reply:
We want to be able to extract the dataset generated by the scan. It's more for user that don't want to directly interact with the scan.